timed/conftest.py

```diff
@@ -1,9 +1,41 @@
+import mockldap
 import pytest
 from django.contrib.auth import get_user_model
 
 from timed.tests.client import JSONAPIClient
 
 
+@pytest.fixture(autouse=True, scope='session')
+def ldap_directory():
+    top = ('o=test', {'o': 'test'})
+    people = ('ou=people,o=test', {'ou': 'people'})
+    groups = ('ou=groups,o=test', {'ou': 'groups'})
+    ldapuser = (
+        'uid=ldapuser,ou=people,o=test', {
+            'uid': ['ldapuser'],
+            'objectClass': [
+                'person', 'organizationalPerson',
+                'inetOrgPerson', 'posixAccount'
+            ],
+            'userPassword': ['Test1234!'],
+            'uidNumber': ['1000'],
+            'gidNumber': ['1000'],
+            'givenName': ['givenName'],
+            'mail': ['ldapuser@example.net'],
+            'sn': ['LdapUser']
+        }
+    )
+
+    directory = dict([top, people, groups, ldapuser])
+    mock = mockldap.MockLdap(directory)
+    mock.start()
+
+    yield
+
+    mock.stop()
+    del mock
```

Comment:
`del mock` shouldn't be necessary, the scope ends here anyway

Reply:
Was part of the example but true... removed it.